### DISCUSSION TITLE
Create community for ImageSharp on Slack

### DISCUSSION THREAD

**Discussion by PeterHevesi** [SixLabors/ImageSharp#2187] [Ideas] [answered]
Hi,
I've recently discovered this awesome library, but I still need help sometimes.
What if you create a community on Slack, which everybody could ask for help and even share some tips & tricks...
That would be sooo perfect :)
I really love, when there is a community for a library or product, that would actually really improve the library or product itself...
Thanks for your fantastic work 😊

**Comment by JimBobSquarePants** [comment, marked as answer]
Hi @PeterHevesi 

You've asked this question in the Discussion forums which are designed for this very purpose. 

We have channels for all community requirements.

<img alt="image" src="https://user-images.githubusercontent.com/385879/180452657-a6cbd2cc-8b79-4778-b1b8-ad9eae369add.png">

A major advantage over using Slack, Discord, or any other chat tool is that the discussions are discoverable via search engines plus deeply integrate into GitHub itself.

We encourage users to contribute to the forum and help each other out.
